### PR TITLE
Feature/ingredient

### PR DESCRIPTION
**Pull request by kristupas-g**

### CODE REVIEW

File: PoSSapi/src/WebUI/Controllers/IngredientController.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    public async Task<ActionResult<Ingredient>> GetIngredientByName([FromQuery] GetIngredientByNameQuery query)
+    {
+        return Ok(await Mediator.Send(query));
+    }
+
+    [HttpGet("list/findByStockStatus")]
+    public async Task<ActionResult<Ingredient>> GetIngredientByName([FromQuery] GetIngredientsByStockStatusQuery query)
+    {
+        return Ok(await Mediator.Send(query));
+    }
```
Comment (Arnasst) [inline PoSSapi/src/WebUI/Controllers/IngredientController.cs:54]: These 3 gets seem like a single endpoint with some query parameters

File: PoSSapi/src/WebUI/Controllers/IngredientController.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    [HttpDelete("{id}")]
+    public async Task<ActionResult> Delete(Guid id)
+    {
+        await Mediator.Send(new DeleteIngredientCommand(id));
+
+        return NoContent();
+    }
+
+    [HttpGet("list")]
+    public async Task<ActionResult<IQueryable<Ingredient>>> GetAllIngredientsPaginatedQuery([FromQuery] GetAllIngredientsQuery query)
```
Comment (Arnasst) [inline PoSSapi/src/WebUI/Controllers/IngredientController.cs:39]: And this should be a paginated list

**Review by Arnasst — CHANGES_REQUESTED** [review]
Previous comments have not been fixed

File: PoSSapi/src/Application/Ingredients/Queries/GetIngredientByNameQuery.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
+using MediatR;
```
Comment (baikauskaite) [inline PoSSapi/src/Application/Ingredients/Queries/GetIngredientByNameQuery.cs:1]: This file doesn't seem to be used anywhere

File: PoSSapi/src/Application/Ingredients/Commands/UpdateIngredientCommandValidator.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+using FluentValidation;
```
Comment (baikauskaite) [inline PoSSapi/src/Application/Ingredients/Commands/UpdateIngredientCommandValidator.cs:1]: Maybe put this file to a `Validators` folder? We're doing this elsewhere in the project.

File: PoSSapi/src/Application/Ingredients/Commands/CreateIngredientCommandValidator.cs [hunk 5]
```diff
@@ -0,0 +1,15 @@
+using FluentValidation;
```
Comment (baikauskaite) [inline PoSSapi/src/Application/Ingredients/Commands/CreateIngredientCommandValidator.cs:1]: Put this file to `Validators` folder?

**Review by baikauskaite — APPROVED** [review]

**Pull request merged**